boards.txt

```diff
@@ -11926,6 +11926,389 @@ adafruit_matrixportal_esp32s3.menu.EraseFlash.none.upload.erase_cmd=
+adafruit_camera_esp32s3.menu.FlashMode.qio120=QIO 120MHz
+adafruit_camera_esp32s3.menu.FlashMode.qio120.build.flash_mode=dio
+adafruit_camera_esp32s3.menu.FlashMode.qio120.build.boot=qio
+adafruit_camera_esp32s3.menu.FlashMode.qio120.build.boot_freq=120m
+adafruit_camera_esp32s3.menu.FlashMode.qio120.build.flash_freq=80m
+adafruit_camera_esp32s3.menu.FlashMode.dio=DIO 80MHz
+adafruit_camera_esp32s3.menu.FlashMode.dio.build.flash_mode=dio
+adafruit_camera_esp32s3.menu.FlashMode.dio.build.boot=dio
+adafruit_camera_esp32s3.menu.FlashMode.dio.build.boot_freq=80m
+adafruit_camera_esp32s3.menu.FlashMode.dio.build.flash_freq=80m
+adafruit_camera_esp32s3.menu.FlashMode.opi=OPI 80MHz
+adafruit_camera_esp32s3.menu.FlashMode.opi.build.flash_mode=dout
+adafruit_camera_esp32s3.menu.FlashMode.opi.build.boot=opi
+adafruit_camera_esp32s3.menu.FlashMode.opi.build.boot_freq=80m
+adafruit_camera_esp32s3.menu.FlashMode.opi.build.flash_freq=80m
+
+adafruit_camera_esp32s3.menu.FlashSize.4M=4MB (32Mb)
+adafruit_camera_esp32s3.menu.FlashSize.4M.build.flash_size=4MB
+
+adafruit_camera_esp32s3.menu.UploadSpeed.921600=921600
+adafruit_camera_esp32s3.menu.UploadSpeed.921600.upload.speed=921600
+adafruit_camera_esp32s3.menu.UploadSpeed.115200=115200
+adafruit_camera_esp32s3.menu.UploadSpeed.115200.upload.speed=115200
+adafruit_camera_esp32s3.menu.UploadSpeed.256000.windows=256000
+adafruit_camera_esp32s3.menu.UploadSpeed.256000.upload.speed=256000
+adafruit_camera_esp32s3.menu.UploadSpeed.230400.windows.upload.speed=256000
+adafruit_camera_esp32s3.menu.UploadSpeed.230400=230400
+adafruit_camera_esp32s3.menu.UploadSpeed.230400.upload.speed=230400
+adafruit_camera_esp32s3.menu.UploadSpeed.460800.linux=460800
+adafruit_camera_esp32s3.menu.UploadSpeed.460800.macosx=460800
+adafruit_camera_esp32s3.menu.UploadSpeed.460800.upload.speed=460800
+adafruit_camera_esp32s3.menu.UploadSpeed.512000.windows=512000
+adafruit_camera_esp32s3.menu.UploadSpeed.512000.upload.speed=512000
+
+adafruit_camera_esp32s3.menu.DebugLevel.none=None
+adafruit_camera_esp32s3.menu.DebugLevel.none.build.code_debug=0
+adafruit_camera_esp32s3.menu.DebugLevel.error=Error
+adafruit_camera_esp32s3.menu.DebugLevel.error.build.code_debug=1
+adafruit_camera_esp32s3.menu.DebugLevel.warn=Warn
+adafruit_camera_esp32s3.menu.DebugLevel.warn.build.code_debug=2
+adafruit_camera_esp32s3.menu.DebugLevel.info=Info
+adafruit_camera_esp32s3.menu.DebugLevel.info.build.code_debug=3
+adafruit_camera_esp32s3.menu.DebugLevel.debug=Debug
+adafruit_camera_esp32s3.menu.DebugLevel.debug.build.code_debug=4
+adafruit_camera_esp32s3.menu.DebugLevel.verbose=Verbose
+adafruit_camera_esp32s3.menu.DebugLevel.verbose.build.code_debug=5
+
+adafruit_camera_esp32s3.menu.EraseFlash.none=Disabled
+adafruit_camera_esp32s3.menu.EraseFlash.none.upload.erase_cmd=
+adafruit_camera_esp32s3.menu.EraseFlash.all=Enabled
+adafruit_camera_esp32s3.menu.EraseFlash.all.upload.erase_cmd=-e
+
+# Adafruit Qualia ESP32-S3 RGB666
+
```

Comment:
Missing line of # to separate boards
```suggestion

##############################################################
# Adafruit Qualia ESP32-S3 RGB666

```

Reply:
done!

Reply:
hmm i see the change in the PR?

Reply:
yes, they showed up now... maybe GH is a bit slow